keyboards/treadstone32/rules.mk

```diff
@@ -1,31 +1 @@
-# MCU name
```

Comment:
Lite wont have the mcu set if this change goes in.

Reply:
6013c72e5 should fix this.